packages/embark-ens/src/index.js

```diff
@@ -75,18 +75,42 @@ class ENS {
     this.modulesPath = dappPath(embark.config.embarkConfig.generationDir, dappArtifacts.symlinkDir);
     this.initated = false;
 
-    this.events.setCommandHandler("ens:resolve", this.ensResolve.bind(this));
-    this.events.setCommandHandler("ens:isENSName", this.isENSName.bind(this));
+    this.events.request("namesystem:node:register", "ens", (readyCb) => {
+      this.events.setCommandHandler("ens:resolve", this.ensResolve.bind(this));
+      this.events.setCommandHandler("ens:isENSName", (name, cb) => {
+        setImmediate(cb, this.isENSName(name));
+      });
 
-    this.embark.events.setCommandHandler("module:namesystem:reset", (cb) => {
-      this.reset();
-      this.init(cb);
+      this.embark.events.setCommandHandler("module:namesystem:reset", (cb) => {
```

Comment:
is this command handler actually used? I can't find any request for it

Reply:
It's used in the tests. It's here: https://github.com/embark-framework/embark/blob/refactor_5_0_0/packages/embark-test-runner/src/test.js#L194

It's used when a test uses different configs so that it can reset using the new config.